frameworks/Dart/start/start-nginx.dockerfile

```diff
@@ -2,4 +2,6 @@ FROM tfb/nginx:latest
 
 COPY ./ ./
 
-CMD ["./start-nginx.sh"]
+RUN ./start-nginx.sh
+
+CMD ["nginx", "-c", "/nginx.conf", "-g", "daemon off;"]
```

Comment:
Nit: I don't think you need the semi-colon

Reply:
You need the semi-colon
`Server start: nginx: [emerg] unexpected end of parameter, expecting ";" in command line`